admin/views/paper-collapsible.php

```diff
@@ -20,26 +20,26 @@
 }
 
 ?>
-<div class="paper tab-block"<?php echo ( $paper_id ) ? ' id="' . esc_attr( 'wpseo-' . $paper_id ) . '"' : ''; ?>>
+<div class="<?php echo esc_attr( 'paper tab-block ' . $class ) ?>"<?php echo ( $paper_id ) ? ' id="' . esc_attr( 'wpseo-' . $paper_id ) . '"' : ''; ?>>
 
 	<?php
 	if ( ! empty( $title ) ) {
 		if ( ! empty( $collapsible ) ) {
 			printf(
-				'<h2 class="help-button-inline"><button type="button" class="toggleable-container-trigger" aria-expanded="%3$s">%1$s <span class="toggleable-container-icon dashicons %2$s" aria-hidden="true"></span></button></h2>',
+				'<button type="button" class="toggleable-container-trigger" aria-expanded="%3$s">%1$s <span class="toggleable-container-icon dashicons %2$s" aria-hidden="true"></span></button>',
```

Comment:
Removing the h2 heading makes these collapsible different from the react ones, more details on the comments thread (with screenshots).